lib/naughty/core.lua

```diff
@@ -583,6 +583,7 @@ function naughty.notify(args)
             die(naughty.notificationClosedReason.dismissedByUser)
         end
     end
+    notification.run = run
```

Comment:
Actually `naughty.notifications` is documented in official wiki: [https://awesomewm.org/doc/api/libraries/naughty.html#notifications](https://awesomewm.org/doc/api/libraries/naughty.html#notifications).